pkg/cli/forward.go

```diff
@@ -137,11 +136,11 @@ func buildForwardKeys(encapOpt bool, commonOpts *skbtrace.CommonOptions) (keys [
 	return
 }
 
-func registerDirectionFlags(flags *pflag.FlagSet, ingressPtr, egressPtr *bool) {
-	flags.BoolVar(ingressPtr, "ingress", false,
-		"Direction is ingress (towards specified interface)")
-	flags.BoolVar(egressPtr, "egress", false,
-		"Direction is egress (from specified interface)")
+func registerDirectionFlags(flags *pflag.FlagSet, inboundPtr, outboundPtr *bool) {
+	flags.BoolVar(inboundPtr, "inbound", false,
+		"Direction is inbound (towards specified interface)")
+	flags.BoolVar(inboundPtr, "outbound", false,
```

Comment:
Looks like it should be `outboundPtr`

Reply:
Yup, it was refactored and fixed in subsequent commit :)